core/src/main/scala/wartremover/warts/ProductOps.scala

```diff
@@ -0,0 +1,40 @@
+package org.brianmckenna.wartremover
+package warts
+
+object ProductOps extends WartTraverser {
+
+  class Op(name: String, error: String) extends WartTraverser {
+    override lazy val className = "org.brianmckenna.wartremover.warts.ProductOps"
+
+    def apply(u: WartUniverse): u.Traverser = {
+      import u.universe._
+
+      val ProductSymbol = rootMirror.staticClass("scala.Product")
+      val Name: TermName = name
+      new u.Traverser {
+        override def traverse(tree: Tree): Unit = {
+          tree match {
+            // Ignore trees marked by SuppressWarnings
+            case t if hasWartAnnotation(u)(t) =>
+            case Select(left, Name) if left.tpe.baseType(ProductSymbol) != NoType ⇒
```

Comment:
You should probably be checking if this method call is synthetic, that is mostly likely the result of your test errors.